### ISSUE TITLE
[MAJOR] Update Versions

### ISSUE THREAD

**Issue by abebeos** [deepnest-io/Deepnest#24]
- (Depends on #31)
- (Processed in parallel with #34)

## Tasks

- [x] attempted VS2022 build (2h), failed, stay with VS2019 for now
- [x] identified and updated first unit
- [x] awaiting budget
- [x] #57
- [x] Version 9371aeb has a broken UI (the hints do not work)
  - [x] check commit after version after v1.1.0 release to find what change breaks it
  - [x] revert d81d05804e052cdaa27feaae0640c318a046c6a5 via  d81d05804e052cdaa27feaae0640c318a046c6a5
- [x] #63
- [x] several updates listed below in the comments via referencing
- [x] update to node 14
- [x] update to node 16 
- [x] #85

### Main Follow-up-Task

-  #83

## Resources / Trials

previous attempts to upgrade

- https://github.com/deepnest-io/Deepnest/issues/34
- #68

**Comment by ruanhuishi** [comment]
I ran the project and selected SVG; But I didn't get the NEST calculation results！

Can you assist in solving the problem?
![企业微信截图_16842240775751](https://github.com/deepnest-io/Deepnest/assets/46099544/3d2e11da-7ec1-41d8-b368-9f1edffe00ed)

**Comment by abebeos** [comment]
> Can you assist in solving the problem?

Please see: https://github.com/deepnest-io/Deepnest/discussions/80#discussioncomment-5914410

**Comment by abebeos** [comment]
@dorkmo , closing this as fulfilled (the electron update is out of scope of this chunk)

- follow-up: #83
